content/documentation/terminology/index.adoc

```diff
@@ -0,0 +1,167 @@
+---
+title: "Terminology"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation"
+---
+
+= Kiali Overview
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+It's a runtime/program that runs in the environment.
+
+== App Name
+
+It's the name of the <<Application>> deployed in your environment.
+
+== App Label
+
+This represents the app label of an object.
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+
+=== Use Case
+
+- Create a Deployment to rollout a ReplicaSet. The ReplicaSet creates Pods in the background. Check the status of the rollout to see if it succeeds or not.
+- Declare the new state of the Pods by updating the PodTemplateSpec of the Deployment. A new ReplicaSet is created and the Deployment manages moving the Pods from the old ReplicaSet to the new one at a controlled rate. Each new ReplicaSet updates the revision of the Deployment.
+- Scale up the Deployment to facilitate more load.
+
+== Destination
+
+It's the place where something is going or being sent.
+
+== DestinationRule
+
+Routing rules correspond to one or more request destination hosts that are specified in a <<VirtualService>> configuration. These hosts may or may not be the same as the actual destination workload and may not even correspond to an actual routable service in the mesh
+
+== Endpoints
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
+
+== Envoy Health
+
+It's the status of the incoming and outcoming traffic.
+
+== Error Rate
+
+it's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
+
+== Gateway
+
+A gateway is a piece of networking hardware used in telecommunications for communications networks that allows data to flow from one discrete network to another.
```

Comment:
Not correct in this context.
It should point to the Gateway term used for Istio in the mesh scenarios:
https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway